packages/cli/src/commands/ui/create/atomic.ts

```diff
@@ -60,17 +62,44 @@ export default class Atomic extends Command {
   }
 
   private async createProject() {
-    return spawnProcess(appendCmdIfWindows`npx`, [
-      `${Atomic.cliPackage}@${flags.version}`,
+    const cfg = this.configuration.get();
+    const authenticatedClient = new AuthenticatedClient();
+    const userInfo = await authenticatedClient.getUserInfo();
+    const apiKey = await authenticatedClient.createImpersonateApiKey(
+      this.args.name
+    );
+
+    const cliArgs = [
+      `${Atomic.cliPackage}@${this.flags.version}`,
+      '--project',
       this.args.name,
-    ]);
+      '--org-id',
+      cfg.organization,
+      '--api-key',
+      apiKey.value!,
+      '--platform-url',
+      platformUrl({environment: cfg.environment}),
+      '--user',
+      userInfo.providerUsername,
+    ];
+
+    return spawnProcess(appendCmdIfWindows`npx`, cliArgs);
```

Comment:
What's the best way of testing this locally, by using verdaccio? I was able to publish locally but I'm getting syntax errors i'm not usually seeing by running `node index.mjs ...args` or `npm start ...args`

Reply:
Yes, Verdaccio is the best way of testing. Do you mean you get errors when running the package as a standalone or via the CLI?

Reply:
Fixed!